lib/pbench/server/api/resources/query_apis/datasets_delete.py

```diff
@@ -79,16 +86,16 @@ def complete(self, dataset: Dataset, params: ApiParams, summary: JSON) -> None:
 
         Args:
             dataset: Dataset object
-            params: API parameters
+            context: CONTEXT dictionary
             summary: summary of the bulk operation
                 ok: count of successful updates
                 failure: count of failures
         """
         # Only on total success we update the Dataset's registered access
         # column; a "partial success" will remain in the previous state.
         if summary["failure"] == 0:
-            self.logger.info("Deleting dataset {} file system representation", dataset)
+            self.logger.debug("Deleting dataset {} file system representation", dataset)
             file_tree = FileTree(self.config, self.logger)
             file_tree.delete(dataset.resource_id)
-            self.logger.info("Deleting dataset {} PostgreSQL representation", dataset)
+            self.logger.debug("Deleting dataset {} PostgreSQL representation", dataset)
```

Comment:
Is it even worth keeping these as `debug()` calls?  Why not just delete them?

Reply:
Fair point.